src/js/jsx/sections/libraries/assets/AssetSection.jsx

```diff
@@ -187,10 +198,14 @@ define(function (require, exports, module) {
                             title={strings.TOOLTIPS.LIBRARY_VIEW_ON_WEBSITE}
                             iconId="libraries-viewonsite"
                             onClick={ui.openURL.bind(null, elementLink)} />
+                         <SplitButtonItem
+                            title={strings.TOOLTIPS.LIBRARY_DELETE}
+                            iconId="delete"
+                            onClick={this._handleDelete} />
                     </SplitButtonList>
                 );
             } else {
-                var subTitle = this.props.subTitle && (<div className="libraries__asset__subtitle">
+                subTitle = this.props.subTitle && (<div className="libraries__asset__subtitle">
```

Comment:
This seems to be the same compared with line 170, we can safely remove these lines.